lib/routes/devices-and-sessions.js

```diff
@@ -307,9 +307,8 @@ module.exports = (log, db, config, customs, push, pushbox, devices) => {
         const sessionToken = request.auth.credentials
         const uid = sessionToken.uid
         const sender = sessionToken.deviceId
-        const ip = request.app.clientAddress
 
-        return customs.checkAuthenticated('invokeDeviceCommand', ip, uid)
+        return customs.checkAuthenticated(request, uid, 'invokeDeviceCommand')
```

Comment:
The order of arguments in `checkAuthenticated` was inconsistent with the other methods on `Customs` object, so I took this opportunity to bring it into alignment.  Passing in the `request` object means it can re-use some shared code from other customs methods.